### ISSUE TITLE
Controls and options aren't announced to screen readers; inaccessible

### ISSUE THREAD

**Issue by downzer0** [select2/select2#1954]
Such hopes for this plugin! Alas, it's not accessible for assistive technology like screen readers. Normal <select> elements are announced so you know what option you're on and which option is selected. This just beeps and the element is "edit blank".

I've forked and plan on making it more accessible....
